packages/net/net_address.pony

```diff
@@ -1,3 +1,5 @@
+type MaybeIPAddr is (U32 | Array[U32])
```

Comment:
I think the one place where this type alias is used doesn't really warrant adding another name to the namespace. I think this line should be removed, and the line below that uses it should just use `(U32 | Array[U32])` directly